tables/output_tables.Rmd

```diff
@@ -154,22 +189,22 @@ telomerase_scores_combined <- bind_rows(telomerase_scores_polya_combined,
 ### Telomerase scores
 ```{r s3 RNA results table p3}
 # get quantiseq immune scores file
-quantiseq_immune <- readRDS("../analyses/immune-deconv/results/quantiseq_deconv-output.rds") %>%
-  dplyr::select(sample, score, cell_type) %>% 
-  dplyr::rename(Kids_First_Biospecimen_ID_RNA = sample) %>%
-  tidyr::spread(cell_type, score)
+quantiseq_immune <- read_rds(quantiseq_file) %>%
+  select(sample, score, cell_type) %>% 
+  rename(Kids_First_Biospecimen_ID_RNA = sample) %>%
+  spread(cell_type, score)
 ```
 
 
 ### combine and output file
 ```{r s3 table combine and output}
 list_s3_table <- list(tp53_scores = tp53_scores,
                       telomerase_scores = telomerase_scores_combined,
-                      quantiseq_immune = quantiseq_immune
+                      quantiseq_immune_deconvolution_fractions = quantiseq_immune
                       )
-openxlsx::write.xlsx(list_s3_table, 
-                     file.path(results_dir, "TableS3-RNA-results-table.xlsx"), 
-                     overwrite=TRUE)
+write.xlsx(list_s3_table, 
+           table_s3_file, 
```

Comment:
The `tp53_scores` and `telomerase_scores` sheets have the same problems as mentioned above.